lib/sinon-chai.js

```diff
@@ -102,10 +102,30 @@
             var sinonMethodName = shouldBeAlways ? alwaysSinonMethod : sinonName;
 
             var messages = getMessages(this._obj, action, nonNegatedSuffix, shouldBeAlways, slice.call(arguments));
+            var passed = this._obj[sinonMethodName].apply(this._obj, arguments);
+            var actual;
+            var expected;
+            if (!passed) {
+                if (sinonMethodName.indexOf("calledWith") === 0) {
```

Comment:
don't you miss the calledOnceWith function?

Reply:
Good catch, I'll update this